Formula/convox.rb

```diff
@@ -0,0 +1,15 @@
+class Convox < Formula
+  desc "The convox AWS PaaS CLI tool"
+  homepage "https://convox.com/"
+  url "https://bin.equinox.io/a/9RUvivxx6Lm/convox-20160606232025-darwin-amd64.tar.gz"
```

Comment:
@DomT4 Ok, I tried modifying the formula to the following:

```diff
diff --git a/Formula/convox.rb b/Formula/convox.rb
index f19137a..6b1a9bd 100644
--- a/Formula/convox.rb
+++ b/Formula/convox.rb
@@ -1,12 +1,16 @@
 class Convox < Formula
   desc "The convox AWS PaaS CLI tool"
   homepage "https://convox.com/"
-  url "https://bin.equinox.io/a/9RUvivxx6Lm/convox-20160606232025-darwin-amd64.tar.gz"
-  version "20160606232025"
-  sha256 "ad9c6cee12b8b34096ee7ce85cd980a5b1784cc05e02df8e1be1ef3ce86a684d"
+  url "https://github.com/convox/rack/archive/20160615213630.tar.gz"
+  version "20160615213630"
+  sha256 "f09b6adda368d67efa87b097299d9cdae1852d1a0c255f6740ff990f0064d937"
+
+  depends_on "go" => :build

   def install
-    bin.install "convox"
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/convox/rack/cmd/convox").install Dir["*"]
+    system "go", "build", "-o", "#{bin}/convox", "-v", "github.com/convox/rack/cmd/convox"
   end

   test do
```

When I tested it locally, I got the following:

```
$ brew install convox
==> Downloading https://github.com/convox/rack/archive/20160615213630.tar.gz
Already downloaded: /Users/michaelwarkentin/Library/Caches/Homebrew/convox-20160615213630.tar.gz
==> go build -o /usr/local/Cellar/convox/20160615213630/bin/convox -v github.com/convox/rack/cmd/convox
Last 15 lines from /Users/michaelwarkentin/Library/Logs/Homebrew/convox/01.go:
2016-06-20 11:34:45 -0400

go
build
-o
/usr/local/Cellar/convox/20160615213630/bin/convox
-v
github.com/convox/rack/cmd/convox

can't load package: package github.com/convox/rack/cmd/convox: no buildable Go source files in /private/tmp/convox-20160620-81124-nh6zcn/rack-20160615213630/src/github.com/convox/rack/cmd/convox

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/Homebrew/homebrew-core/issues

These open issues may also help:
convox 20160606232025 (new formula) https://github.com/Homebrew/homebrew-core/pull/1987
```

However, if I run the command shown in those logs manually, it seems to build correctly. Any ideas?

```
$ ll /usr/local/Cellar/convox/20160615213630/bin/convox
ls: /usr/local/Cellar/convox/20160615213630/bin/convox: No such file or directory

$ go build -o /usr/local/Cellar/convox/20160615213630/bin/convox -v github.com/convox/rack/cmd/convox
github.com/convox/rack/vendor/github.com/go-ini/ini
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/client/metadata
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/awserr
github.com/convox/rack/vendor/github.com/jmespath/go-jmespath
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/private/endpoints
github.com/convox/rack/vendor/github.com/cheggaaa/pb
github.com/convox/rack/vendor/github.com/codegangsta/cli
github.com/convox/rack/cmd/convox/templates
github.com/convox/rack/vendor/github.com/mattn/go-isatty
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/credentials
github.com/convox/rack/vendor/github.com/shiena/ansicolor
github.com/convox/rack/vendor/github.com/fatih/color
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/parsers
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/ulimit
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/volume
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/Sirupsen/logrus
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/opts
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/ioutils
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/promise
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/system
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/awsutil
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/opencontainers/runc/libcontainer/user
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/pools
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/request
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/fileutils
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/stdcopy
github.com/convox/rack/vendor/gopkg.in/yaml.v2
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/archive
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/homedir
github.com/convox/rack/vendor/golang.org/x/crypto/ssh/terminal
github.com/convox/rack/vendor/golang.org/x/net/websocket
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/client
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/corehandlers
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/ec2metadata
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/private/protocol
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/private/protocol/rest
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/private/protocol/query/queryutil
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/credentials/ec2rolecreds
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/private/protocol/xml/xmlutil
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/defaults
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/private/signer/v4
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/private/protocol/query
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/private/waiter
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/aws/session
github.com/convox/rack/vendor/github.com/fsouza/go-dockerclient
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/service/cloudformation
github.com/convox/rack/client
github.com/convox/rack/vendor/github.com/briandowns/spinner
github.com/convox/rack/vendor/github.com/jehiah/go-strftime
github.com/convox/rack/vendor/github.com/xtgo/uuid
github.com/convox/rack/vendor/github.com/segmentio/analytics-go
github.com/convox/rack/vendor/github.com/stvp/rollbar
github.com/convox/rack/cmd/convox/stdcli
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/private/protocol/restxml
github.com/convox/rack/vendor/github.com/docker/docker/builder/dockerignore
github.com/convox/rack/vendor/github.com/Sirupsen/logrus
github.com/convox/rack/vendor/github.com/dustin/go-humanize
github.com/convox/rack/vendor/github.com/aws/aws-sdk-go/service/s3
github.com/convox/rack/vendor/github.com/equinox-io/equinox/internal/go-update/internal/binarydist
github.com/convox/rack/vendor/github.com/equinox-io/equinox/internal/go-update/internal/osext
github.com/convox/rack/vendor/github.com/docker/docker/pkg/fileutils
github.com/convox/rack/vendor/github.com/equinox-io/equinox/internal/go-update
github.com/convox/rack/vendor/github.com/equinox-io/equinox/internal/osext
github.com/convox/rack/api/manifest
github.com/convox/rack/vendor/github.com/equinox-io/equinox/proto
github.com/convox/rack/vendor/github.com/mitchellh/go-homedir
github.com/convox/rack/vendor/github.com/equinox-io/equinox
github.com/convox/rack/vendor/github.com/convox/version
github.com/convox/rack/cmd/convox

$ ll /usr/local/Cellar/convox/20160615213630/bin/convox
-rwxr-xr-x  1 michaelwarkentin  staff  15924924 20 Jun 11:37 /usr/local/Cellar/convox/20160615213630/bin/convox

$ /usr/local/Cellar/convox/20160615213630/bin/convox
convox: command-line application management

Usage:
  convox <command> [args...]

Subcommands: (convox help <subcommand>)
  api		api endpoint
  apps		list deployed apps
  ...
```